platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/activity/EspressoActivityTest.java

```diff
@@ -0,0 +1,24 @@
+package com.mapbox.mapboxsdk.testapp.activity;
+
+import android.support.test.espresso.IdlingResource;
+import com.mapbox.mapboxsdk.testapp.activity.espresso.EspressoTestActivity;
+import com.mapbox.mapboxsdk.testapp.utils.LoadStyleIdlingResource;
+
+/**
+ * Base class for all tests using EspressoTestActivity as wrapper.
+ * <p>
+ * Uses {@link LoadStyleIdlingResource} to load "assets/streets.json" as style.
+ * </p>
+ */
+public class EspressoActivityTest extends BaseActivityTest {
```

Comment:
Can we rename `EspressoActivityTest` to `EspressoTest`? Similarly with `BaseActivityTest`. It's a little bit hard to read and distinguish between `EspressoTestActivity`.